cmd/archive_upload_server/server.go

```diff
@@ -40,23 +40,27 @@ const (
 
 var (
 	port   = os.Getenv("PORT")
-	bucket = flag.String("bucket", "routeviews-archives", "Cloud storage bucket name.")
+	bucket = flag.String("bucket", "routeviews-archives",
+		"GCS bucket for MRT dumps to be converted.")
+	unconvertedBucket = flag.String("unconverted_bucket", "routeviews-unconverted",
```

Comment:
I was thinking we may want 1 bucket for RIBS, and one for Validator JSON output, and later one maybe for RIS data (though that looks a heck of a lot like routeviews data... so maybe not?)

in the end it looks (to me) like a map of:
  pb.FileRequest_<type> -> bucket
  pb.FileRequest_ROUTEVIEWS
might do us better here? It's also possible that we should split the 
  pb.FileRequest_ROUTEVIEWS
  
into something like:
 pb.FileRequest_ROUTEVIEWS_UPDATES
 pb.FileRequest_ROUTEVIEWS_RIB

or maybe there's a better way to handle this?

Reply:
From our chat, it looks like ripe gets this sort of file per collector:
          ...
 	updates.20220401.0000.gz	2022-04-01 00:05	617K	 
 	bview.20220401.1600.gz	2022-04-01 16:37	129M	 
 	bview.20220401.0800.gz	2022-04-01 08:40	129M	 


The updates files look like RV update type files, the bview files are (we think) ribdumps:
$ wget https://data.ris.ripe.net/rrc05/2022.04/bview.20220401.1600.gz
$ gzip -dc bview.20220401.1600.gz > foo
$ ~/scripts/git/bgpdump/bgpdump -M foo |more
2022-04-01 14:33:27 [info] logging to syslog
TABLE_DUMP2|04/01/22 16:00:00|B|193.203.0.192|47692|0.0.0.0/0|47692 3356|IGP
TABLE_DUMP2|04/01/22 16:00:00|B|2001:7f8:30::2:1:4:7692|47692|::/0|47692 3356|IGP
TABLE_DUMP2|04/01/22 16:00:00|B|193.203.0.157|35369|1.0.0.0/24|35369 13335|IGP
TABLE_DUMP2|04/01/22 16:00:00|B|193.203.0.182|48362|1.0.0.0/24|48362 13335|IGP
TABLE_DUMP2|04/01/22 16:00:00|B|193.203.0.19|47147|1.0.0.0/24|47147 13335|IGP
...

So, probably we either rely on filenames and directory to save us, or we pick a new enum value for 'ripe dump' vs 'rv dump'.

Reply:
I've updated the PR with the following changes:
- Redirect files from different projects to different buckets.
- Read the `pb.FileRequest_Project` to GCS bucket relationship from a YAML config file.
- In upload clients, allow insecure gRPC connections for local testing. 

PTAL.